SearchCriteria/Books/AuthorSearchCriteria.cs

```diff
@@ -0,0 +1,19 @@
+﻿
```

Comment:
Delete this space, just like you did in the other classes, good job.

Reply:
Already resolved, thanks